proto/vendor/share/shwap/pb/shwap.proto

```diff
@@ -35,9 +35,9 @@ message RowShares {
 }
 
 message RangeNamespaceData {
```

Comment:
Changes here (and RowRootProof, Proof below) seem breaking, but it's not used anywhere 🤷

Reply:
we're using proofs from `celestia.core.v1.proofs`
https://github.com/eigerco/lumina/pull/707/files/f4261e9620f0ccca6f43409bd522194e0cbd27ad#diff-52625f6b3739440305b89ea44ce599406413264f35815729945f77c9886d6ad9L8